packages/navy/src/util/__tests__/nipio.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-env mocha */
+
+import {expect} from 'chai'
+import {getNIPSubdomain, getHostForService} from '../nipio'
+
+describe('nipio', function () {
+
+  describe('getNIPSubdomain', function () {
+
+    it('should return a short nip.io domain when IP is 127.0.0.1', async function () {
```

Comment:
"should return a short nip.io domain when IP is 127.0.0.1" -> "should return the correct nip.io domain when IP is 127.0.0.1"

`127.0.0.1` should no longer be considered "short"